client/src/components/qa/QuestionList.jsx

```diff
@@ -4,41 +4,38 @@ import axios from 'axios';
 import Question from './Question.jsx';
 import AddQuestion from './AddQuestion.jsx';
 
-function QuestionList({ questionList, getQuestions }) {
+function QuestionList({
+  questionList, getQuestions, product, toggleAddQuestion, questOpen, setQuestOpen,
+}) {
   const [questionCount, setQuestionCount] = useState(2);
 
-  if (questionCount >= questionList.length) {
-    return (
-      <div id="QuestionList">
-        {questionList.slice(0, questionCount).map((question) => (
-          <Question
-            body={question.question_body}
-            helpfulness={question.question_helpfulness}
-            id={question.question_id}
-            key={question.question_id}
-            answers={question.answers}
-            getQuestions={getQuestions}
-          />
-        ))}
-      </div>
-    );
-  }
   return (
     <div id="QuestionList">
       {questionList.slice(0, questionCount).map((question) => (
         <Question
+          key={question.question_id}
           body={question.question_body}
```

Comment:
I have a question, why don't you just pass the whole question object as a prop to your Question component? Is the object very big and bulky and it doesn't make sense to pass it all, so you pass only relevant properties?

Reply:
Hmm it was more organizational than anything as when I was writing that portion I wasn't sure exactly what I needed so I was specifying them to make the props more clear I guess?. You're totally right though, I can just pass the whole question object as a prop and it'd be fine. I'll look to change that as it'd be much cleaner.